src/javascript/_common/geocoder.js

```diff
@@ -1,64 +1,106 @@
 /* global google */
 const scriptjs           = require('scriptjs');
-const localize           = require('./localize').localize;
+const getElementById     = require('./common_functions').getElementById;
 const applyToAllElements = require('./utility').applyToAllElements;
-const createElement      = require('./utility').createElement;
 const Client             = require('../app/base/client');
 
 const Geocoder = (() => {
     let el_btn_validate,
-        el_error;
+        el_error,
+        el_geocode_status,
+        el_loader,
+        el_postcode_row,
+        el_success,
+        is_el_states_select;
```

Comment:
I think `is_states_el_select` is more accurate, since we're checking if the states is `select` element.